<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sign_up_window</class>
 <widget class="QMainWindow" name="sign_up_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix2/smallized/sign_up copy.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>241</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
font: 700 24pt &quot;Roboto&quot;;
background-image: url(:/new/prefix2/smallized/tranc.png);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>218</y>
      <width>241</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
font: 700 24pt &quot;Roboto&quot;;
background-image: url(:/new/prefix2/smallized/tranc.png);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>241</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
font: 700 24pt &quot;Roboto&quot;;
background-image: url(:/new/prefix2/smallized/tranc.png);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>350</y>
      <width>241</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
font: 700 24pt &quot;Roboto&quot;;
background-image: url(:/new/prefix2/smallized/tranc.png);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>420</y>
      <width>241</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
font: 700 24pt &quot;Roboto&quot;;
background-image: url(:/new/prefix2/smallized/tranc.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>498</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
    background-color: transparent;



background-image: url(:/new/prefix2/smallized/tranc.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>380</y>
      <width>401</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 0);
background-image: url(:/new/prefix2/smallized/tranc.png);
font: 700 9pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>Registraion Complete</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>380</y>
      <width>401</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-image: url(:/new/prefix2/smallized/tranc.png);
font: 700 9pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>Registraion Faild</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
